app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/AdvancedPricing.php

```diff
@@ -574,7 +574,7 @@ private function specialPriceDataToInline()
                 $this->meta,
                 [
                     'label' => __('Special Price From'),
-                    'additionalClasses' => 'admin__control-grouped-date',
+                    'additionalClasses' => 'admin__control-grouped-date admin__control-grouped',
```

Comment:
Additional classes can be array, you don't need to concatenate strings

Reply:
there is no concatenation. Please let me know if I need to use array.